src/Queue.php

```diff
@@ -103,13 +103,17 @@ public function push(JobInterface $job): string
 
     /**
      * Execute all existing jobs and exit
+     *
+     * @param int $max
      */
-    public function run(): void
+    public function run(int $max = 0): void
```

Comment:
Would be nice to check that `$max` greater than zero or equals to zero

Reply:
In the current implementation it doesn't matter if `$max` is zero or less then zero.